### PR TITLE
[razor-network-voting] update strategy

### PR DESCRIPTION
**Pull request by rajkharvar**
Changes proposed in this pull request:
- Update network and addresses in `examples.json`.
- Update strategy to support paginated data.

### CODE REVIEW

File: src/strategies/razor-network-voting/index.ts [hunk 1]
```diff
@@ -34,43 +97,10 @@ export async function strategy(
...
-    // @ts-ignore
-    params.delegators.__args.block = { number: snapshot };
-  }
-
   const score = {};
 
-  // subgraph request 1 : it fetches all the details of the stakers and delegators.
-  const result = await subgraphRequest(RAZOR_NETWORK_SUBGRAPH_URL, params);
+  // subgraph request : it fetches all the details of the stakers and delegators.
+  const result = await getAllData(snapshot);
```
Comment (ChaituVR) [inline src/strategies/razor-network-voting/index.ts:103]: How many network requests it can send? 😅

Comment (ChaituVR) [reply src/strategies/razor-network-voting/index.ts:103]: Ideally, it should not send more than 5 requests https://github.com/snapshot-labs/snapshot-strategies#checklist-for-adding-a-new-voting-strategy-or-validation-strategy

Comment (rajkharvar) [reply src/strategies/razor-network-voting/index.ts:103]: It's highly unlikely that we will surpass the upper limit of more than five requests.

Comment (ChaituVR) [reply src/strategies/razor-network-voting/index.ts:103]: Then it should be cool :)

**Pull request merged**